data/behavior_data.c

```diff
@@ -66,6 +66,7 @@
 #define BC_HH(a, b) (_SHIFTL(a, 16, 16) | _SHIFTL(b, 0, 16))
 #define BC_W(a) ((uintptr_t)(u32)(a))
 #define BC_PTR(a) ((uintptr_t)(a))
+#define BC_BPTR(a, b) (_SHIFTL(a, 24, 8) + OS_K0_TO_PHYSICAL(b))
```

Comment:
I think `|` is better to use than `+` because it is later extracted using `& 0xffffff`, not `-`.

Reply:
| is not possible to use here, will cause compiler error. The linker can only handle constant additions / subtractions to symbols, not any more complex expression than that.